demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -1534,6 +1534,9 @@ export class DemoMeetingApp
       case 'videoInputSelected':
       case 'audioInputUnselected':
       case 'videoInputUnselected':
+      case 'meetingReconnected':
+      case 'receivingAudioDropped':
+      case 'signalingDropped':
       case 'attendeePresenceReceived': {
         // Exclude the "meetingHistory" attribute for successful events.
```

Comment:
This is not part of your change but since we are adding lifecycle events here, you can change this comment with "successful" -> "published" may be.